app/screens/staking-delegation-amount-input-screen/staking-delegation-amount-input-screen.tsx

```diff
@@ -60,7 +62,9 @@ export class StakingDelegationAmountInputScreen extends React.Component<StakingD
   }
 
   private onPressNextButton = async () => {
+    logAnalyticsEvent('DelegateClickBound')
     if (await this.createTransactionForSigning()) {
+      logAnalyticsEvent('DelegateCreateBoundTx')
```

Comment:
`Bound` is very ambiguous 

How about `StakingDelegateConfirmAccount` and `StakingDelegateCreateTxForSigning`

Reply:
the plan was to name all `Delegate` related event with `Delegate` as beginning, then `bound` `unbound` `reward` as subclass
Also we have a event name constraint of 40 char, so maybe need to think of something short

Reply:
As you can see, I have already named the screen prefix as Staking

I am following the RPC from Cosmos
https://cosmos.network/rpc/#/ICS21

So you may follow, the subclass will be
- Delegate
- Undelegate
- RewardsWithdraw

Reply:
I think Withdraw is ambiguos, how about shortening `Rewards` to `Rwd`

StakeDelegate
StakeUndelegate
StakeWithdrawRwd